api/src/routers/user.py

```diff
@@ -1,3 +1,70 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends, HTTPException
+from sqlmodel import Session, select
+from src.db import get_session
+from src.models import (
+    Experience,
+    User,
+    UserCreatePublic,
+    UserPublic,
+    UserUpdateExperience,
+)
 
 router = APIRouter()
+
+
+@router.get("/user/{user_id}")
+async def get_user(
+    user_id: int,
+    session: Session = Depends(get_session),  # noqa: B008
+) -> UserPublic:
+    user = session.exec(select(User).where(User.user_id == user_id)).first()
+
+    if user is None:
+        raise HTTPException(404, "User not found")
+    return user
+
+
+@router.post("/user/{user_id}")
```

Comment:
POST to `/user` or event a `/user/create` would be the more ideal method. See https://github.com/FloncDev/exquisite-exoplanets/pull/32#discussion_r1685795869

Reply:
Very true, something for v3!